README.md

```diff
@@ -13,17 +13,45 @@
 ------
 
 ## Build
-The project is compiled using `gradle` but for the lazy ones we provide a simple `Makefile`:
+The project is compiled using `gradle` but for the we also provide a simple `Makefile` for basic usage:
 ```shell
 # Compile and start the editor
 make run
 ```
 
 ## Install
+### Linux
+We provide `.deb` and `.rpm` packaging formats directly from the github releases, but you can also install via `flatpak`.
+
+#### Flatpak
+```shell
+flatpak install sillydan1/graphedit/graphedit
```

Comment:
Comment this out for now. We can reintroduce once a release is actually installavle through flatpak